packages/kuma-gui/src/app/App.vue

```diff
@@ -103,4 +119,12 @@ router.afterEach(() => {
 .logo {
   max-height: 36px;
 }
+
+:deep(.app-sidebar) {
+  --icon-home: url('@/assets/images/navigation/icon-home.svg');
+  --icon-zones: url('@/assets/images/navigation/icon-zones.svg');
+  --icon-meshes: url('@/assets/images/navigation/icon-meshes.svg');
+  --icon-configuration: url('@/assets/images/navigation/icon-configuration.svg');
+  --icon-zone-egresses: url('@/assets/images/navigation/icon-zone-egresses.svg');
+}
```

Comment:
Just to leave some sort of note on what we discussed yesterday:

Ideally whilst we have an `XIcon` component we should try and use it for all our icons, instead of "all our Iconography uses XIcon... apart from this little bit over there"

Personally, I would rather all our icongraphy used CSS, its DOM node friendly, more accessible and more flexible. But not something we are in a position to change right now tho.

---

That said, lets stick with what you have here for now, not all of these icons are available as kong/icons, but just to make a note that we may circle back and change it to use XIcon here at some point (or the other way around and make all our icons use CSS). i.e. how to do iconography should be consistent everywhere.

Reply:
Yes, sounds good 👍 🙂